tests/js/web/PulsarFormComponentTest.js

```diff
@@ -259,7 +259,7 @@ describe('PulsarFormComponent', function() {
         it('should initiate the date pickers', function () {
             this.pulsarForm.refresh();
 
-            expect($.fn.pikaday).to.have.been.calledOnce;
+            expect($.fn.pikaday).to.have.been.called;
```

Comment:
Can we remain explicit on the call count here in order to make sure we aren't ever calling it an unexpected amount of times?

With sinon we have `calledOnce`, `calledTwice`, `calledThrice` and `callCount` (that returns the call count as a number)